platform/src/components/aws/dynamo.ts

```diff
@@ -616,11 +614,12 @@ export class Dynamo extends Component implements Link.Linkable {
     const sourceName = this.constructorName;
 
     // Validate stream is enabled
-    if (!this.isStreamEnabled)
-      throw new Error(
+    if (!this.nodes.table.streamEnabled.apply((streamEnabled) => {
+      if (!streamEnabled) throw new Error(
         `Cannot subscribe to "${sourceName}" because stream is not enabled.`,
       );
-
+    }))
+    
```

Comment:
we should probably do something like this as we do in other parts of the codebase:


```ts
    return all([this.nodes.table.streamEnabled]).apply(([streamEnabled]) => {
      if (!streamEnabled) {
        throw new Error(`Cannot subscribe to "${sourceName}" because stream is not enabled.`);
      }
      
      return isFunctionSubscriber(...
    }))
    
```